docs/website/data/integrations.yaml

```diff
@@ -1011,6 +1011,29 @@ integrations:
       - https://github.com/EliuX/flask-opa
     blogs:
       - https://github.com/EliuX/flask-opa/tree/master/examples
+ 
+  permit:
+    title: Permit.io
+    description:  |
+      Permit.io empowers developers to bake in permissions and access-control into any product in minutes and takes away the pain of constantly rebuilding them.
+      Permit is based on OPA.
+    labels:
+      category: authorization
+      layer: application
+    inventors:
+      - permitio
+    software:
+      - permit
```

Comment:
Let's remove those two lines -- they only matter when your having an extension to something... e.g. linking to envoy for opa-envoy-plugin.

Reply:
removed